test-zone-data/Address-TP/address03/README.md

```diff
@@ -0,0 +1,179 @@
+# Address03

```

Comment:
After review comment I renamed the equivalent of this file for DNSSEC10 to "test-zones-output.md" and added another README.md of more normal type. Would that work?

https://github.com/zonemaster/zonemaster/blob/master/test-zone-data/DNSSEC-TP/dnssec10/test-zones-output.md

Reply:
Sure, I’ve fixed that.